src/games/new_york_times.ts

```diff
@@ -0,0 +1,88 @@
+import { GameBuilder } from '../core/builders/game_builder.js';
+import { MatchType } from '../core/message_parser.js';
+import { seconds_to_display_time } from '../util.js';
+
+export const Wordle = new GameBuilder('Wordle')
+  .set_matcher(/Wordle (\d+\s?,?\d+) ([1-6X]\/6)/, [
+    MatchType.Day,
+    MatchType.Score,
+  ])
+  .set_responder(
+    (entry) =>
+      `${entry.user.server_name ?? entry.user.name} scored ${entry.score} on Wordle ${entry.day_id}`
+  )
+  .build();
+
+export const Connections = new GameBuilder('Connections')
+  .set_matcher(/Connections\s\sPuzzle\s#(\d+)\s([🟩🟨🟦🟪\s]+)/u, [
+    MatchType.Day,
+    MatchType.Score,
+  ])
+  .set_score_parser((match) =>
+    match
+      .split('\n')
+      .reduce(
+        (sum, line) =>
+          (sum +=
+            new Set(
+              line
+                .trim()
+                .split('\uD83D')
+                .filter((n) => n !== '')
+            ).size == 1
+              ? 0
+              : 1),
+        0
+      )
+      .toString()
```

Comment:
Not sure I agree with the formatting choice 😅

Reply:
Can't find a Prettier config to stop it from doing this 😭